projects/kit/components/line-clamp/line-clamp.directive.ts

```diff
@@ -39,6 +39,7 @@ export class TuiLineClampDirective {
     ) {
         if (
             cssRef.supports('-webkit-line-clamp', '1') ||
+            !documentRef.createRange ||
```

Comment:
I think just in case

`!documentRef?.createRange`

Reply:
No, here we always have document.